ubuntu.json

```diff
@@ -180,7 +180,7 @@
     "vagrantfile_template": "",
     "version": "0.1.0",
     "virtualbox_guest_os_type": "Ubuntu_64",
-    "vm_name": "ubuntu1804",
-    "vmware_guest_os_type": "ubuntu-64"
+    "vm_name": "ubuntu2004",
+    "vmware_guest_os_type": "ubuntu64Guest"
```

Comment:
@fasmat I actually created a new VM with VMware Fustion 11.5.3 for verifying this, but found that in the .vmx it still says `guestOS = "ubuntu-64"`

Are you sure `"ubuntu64Guest"` is correct here?

Reply:
I used the information from this [vmware documenation page](https://vdc-download.vmware.com/vmwb-repository/dcr-public/da47f910-60ac-438b-8b9b-6122f4d14524/16b7274a-bf8b-4b4c-a05e-746f2aa93c8c/doc/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html) and this [discussion thread](https://discuss.hashicorp.com/t/list-of-valid-guest-os-type-values-for-vsphere-builder/4671/15).

You are correct, a manually created VMWare Fusion VM has "ubuntu-64" as guestOS type and there is no need to change it, so I'm reverting this.